Firestore/core/src/firebase/firestore/local/query_data.cc

```diff
@@ -85,10 +85,16 @@ QueryData QueryData::Invalid() {
                    SnapshotVersion(SnapshotVersion::None()), {});
 }
 
-QueryData QueryData::Copy(SnapshotVersion snapshot_version,
-                          ByteString resume_token,
-                          model::ListenSequenceNumber sequence_number) const {
+QueryData QueryData::WithSequenceNumber(
+    model::ListenSequenceNumber sequence_number) const {
   return QueryData(Query(query_), target_id_, sequence_number, purpose_,
+                   snapshot_version_, resume_token_);
+}
+
+QueryData QueryData::WithResumeToken(
+    nanopb::ByteString resume_token,
```

Comment:
There are already `using` statements for `ByteString` and `SnapshotVersion`. You can use these unqualified in the .cc file (but not in the header).

Also consider adding a `using` statement for `ListenSequenceNumber` and `TargetId` and shortening those too.

Reply:
Done (added using statements for all the types you mentioned).